api/v1beta1/grafanaalertrulegroup_types.go

```diff
@@ -201,6 +201,10 @@ func (in *GrafanaAlertRuleGroup) CommonStatus() *GrafanaCommonStatus {
 	return &in.Status
 }
 
+func (in *GrafanaAlertRuleGroup) NamespacedResource() NamespacedResource {
+	return NewNamespacedResource(in.Namespace, in.Name, in.GroupName())
```

Comment:
Is there an advantage of using something other than a UID in the NamespacedResource?
(applies to other resources as well)

Just so it's nicer to look at in the output?

My main "issue" is that non-UID identifiers could contain `/` which would mess with the splitting logic

Reply:
`AlertRuleGroups` don't have a UID, only the rules themselves.
The same is true for `MuteTimings` and `NotificationTemplates`.
Hence I don't have a UID to use 😅